DataFormats/HGCalRecHit/BuildFile.xml

```diff
@@ -0,0 +1,11 @@
+<use name="eigen"/>
+<use name="rootcore"/>
+<use name="DataFormats/Common"/>
+<use name="DataFormats/Portable"/>
+<use name="DataFormats/DetId"/>
+<use name="DataFormats/ForwardDetId"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
+<flags ALPAKA_BACKENDS="cuda rocm"/>
```

Comment:
```suggestion
<flags ALPAKA_BACKENDS="!serial"/>
```
is preferred

Reply:
Done in https://github.com/cms-sw/cmssw/pull/45665/commits/f9e6cfac0101ac0d38179e820c185c91852f9618